samples/bluetooth/shell_bt_nus/README.rst

```diff
@@ -17,31 +17,42 @@ Requirements
 
 * One of the following development boards:
 
-  * |nRF5340DK|
-  * |nRF52840DK|
-  * |nRF52DK|
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set1_start
+   :end-before: set1_end
 
 * A second nRF52 Development Kit board (PCA10040) for connecting with bt_nus_shell.py.
-  Alternatively, you can use :ref:`ble_console_readme` for connecting (Linux only).
+  Alternatively, you can use :ref:`ble_console_readme` for connecting, using Linux only.
 
 
 Building and running
 ********************
 
-This sample can be found under :file:`samples/bluetooth/shell_bt_nus` in the
-|NCS| folder structure.
+This sample can be found under :file:`samples/bluetooth/shell_bt_nus` in the |NCS| folder structure.
```

Comment:
```suggestion
.. |sample path| replace:: :file:`samples/zigbee/light_bulb`
```

Reply:
Hey @greg-fer, are you sure about this suggestion? The original path to the sample seems to be correct.

Reply:
Sorry, forgot to update the path in the suggestion. See the source of other samples -- this formula is standard for most of them.

Reply:
fixed